apps/modernization-ui/src/SkipLink/SkipLinkContext.tsx

```diff
@@ -24,14 +26,30 @@ interface SkipLinkProviderProps {
 
 export const SkipLinkProvider = ({ children }: SkipLinkProviderProps) => {
     const [currentFocusTarget, setCurrentFocusTarget] = useState('');
+    const [previousFocusTarget, setPreviousFocusTarget] = useState('');
 
     const skipTo = (id: string) => {
         setCurrentFocusTarget(id);
     };
 
+    const previousTarget = (id: string) => {
+        setPreviousFocusTarget(id);
+    };
+
+    useEffect(() => {
+        const handleKeyDown = (event: KeyboardEvent) => {
```

Comment:
question: is `ALT-X` a global shortcut or is it just for searches?

Reply:
just for searches for now. Event and Patient search,
It should only work on that page or another page that has the same buttons and function if there is another that works like the search page which in this case can be event search

Reply:
Since this is just for the search pages shouldn't the functionality be on the search pages and not the skip link?  

The `alt-x` shortcut is specific to search features to allow a quick way to return to the criteria and is a distinct feature from the skip link.  In this case the target of the skip link is the same as the target for `alt-x` however, it doesn't mean the shortcut is part of the skip link feature.